### PR TITLE
Update getting started doc

### PR DESCRIPTION
**Pull request by cacheung**
Update getting started doc.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Documentation/getting-started.md [hunk 1]
```diff
@@ -1,11 +1,25 @@
-# Getting Started with Analytics SDK
+# Adobe Analytics Android Extension
 
-## Before starting
+The [`Analytics`](https://developer.adobe.com/client-sdks/documentation/adobe-analytics/) extension represents the Analytics Adobe Experience Platform SDK that is required for sending mobile interaction data from your app to Adobe Analytics.
```
Comment (kevinlind) [inline Documentation/getting-started.md:3]: As this is already a hyperlink, remove the backticks.

File: Documentation/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# AEP Analytics SDK Documentation
+# Adobe Analytics Extension Documentation
```
Comment (addb) [inline Documentation/README.md:1]: Adobe Experience Platform - Analytics extension for Android

refer:https://github.com/adobe/aepsdk-media-android/tree/staging

File: code/testapp/build.gradle [hunk 3]
```diff
@@ -52,7 +52,6 @@ dependencies {
     implementation 'com.adobe.marketing.mobile:identity:2.+'
     implementation 'com.adobe.marketing.mobile:assurance:2.+'
 
-
```
Comment (kevinlind) [inline code/testapp/build.gradle:55]: This file was recently updated by another PR. You can remove this change and merge in the latest from `dev` branch.

File: Documentation/getting-started.md [hunk 4]
```diff
@@ -1,11 +1,25 @@
-# Getting Started with Analytics SDK
+# Adobe Analytics Android Extension
```
Comment (addb) [inline Documentation/getting-started.md:1]: Getting Started with Adobe Analytics Android Extension

File: Documentation/getting-started.md (hunk 1 above)
Comment (addb) [inline Documentation/getting-started.md:3]: The ~~[`Analytics`](https://developer.adobe.com/client-sdks/documentation/adobe-analytics/) represents the Analytics~~ [Adobe Experience Platform ~~SDK~~ - Analytics Extension](https://developer.adobe.com/client-sdks/documentation/adobe-analytics) is required for sending ~~mobile~~ interaction data from the mobile application to Adobe Analytics.

File: README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# aepsdk-analytics-android
+# Adobe Analytics Android Extension
```
Comment (addb) [inline README.md:1]: Adobe Experience Platform - Analytics extension for Android

File: Documentation/api-reference.md [hunk 6]
```diff
@@ -134,22 +166,20 @@ Analytics.getQueueSize(object: AdobeCallbackWithError<Long> {
 
 ---
 
-## getTrackingIdentifier
+### getTrackingIdentifier
 
 > ℹ️ Before using this API, see [Identify unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html).
```
Comment (addb) [inline Documentation/api-reference.md:171]: remove: ℹ️
`> Note: `

File: Documentation/api-reference.md [hunk 7]
```diff
@@ -166,6 +196,13 @@ Analytics.getTrackingIdentifier(new AdobeCallbackWithError<String>() {
...
+##### Syntax
+```Kotlin
+fun getTrackingIdentifier(callback: AdobeCallback<String?>)
+```
+
+##### Example
+
 ```Kotlin
 Analytics.getTrackingIdentifier(object: AdobeCallbackWithError<String> {
     override fun call(id: String?) {
```
Comment (addb) [inline Documentation/api-reference.md:208]:
```
override fun call(aid: String?) {
//Handle the tracking identifier (AID)
}
```

File: Documentation/api-reference.md [hunk 8]
```diff
@@ -134,22 +166,20 @@ Analytics.getQueueSize(object: AdobeCallbackWithError<Long> {
...
 ```
 
-### Example
-
-#### Java
-
+##### Example
 ```Java
 Analytics.getTrackingIdentifier(new AdobeCallbackWithError<String>() {
     @Override
```
Comment (addb) [inline Documentation/api-reference.md:185]:
```
public void call(String aid) {
//Handle the tracking identifier (AID)
}
```

File: Documentation/api-reference.md [hunk 9]
```diff
@@ -180,22 +217,20 @@ Analytics.getTrackingIdentifier(object: AdobeCallbackWithError<String> {
 
 ---
 
-## getVisitorIdentifier
+### getVisitorIdentifier
 
 > Before use this API, see [Identify unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html).
```
Comment (addb) [inline Documentation/api-reference.md:222]: add` > Note: ` for special github formatting with icon

> **Note**
> Before use this API, see [Identify unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html).

File: Documentation/api-reference.md [hunk 10]
```diff
@@ -226,54 +266,62 @@ Analytics.getVisitorIdentifier(object: AdobeCallbackWithError<String> {
 
 ---
 
-## sendQueuedHits
+### sendQueuedHits
 
 Sends all queued hits in the offline queue to Analytics, regardless of the current hit batch settings.
```
Comment (addb) [inline Documentation/api-reference.md:271]: `Sends all the queued hits ...`

File: Documentation/api-reference.md [hunk 11]
```diff
@@ -226,54 +266,62 @@ Analytics.getVisitorIdentifier(object: AdobeCallbackWithError<String> {
...
+#### Java
 
+##### Syntax
 ```Java
 public static void sendQueuedHits()
 ```
 
-### Example
-
-#### Java
```
Comment (addb) [inline Documentation/api-reference.md:241]: Add this language header back

File: Documentation/api-reference.md [hunk 12]
```diff
@@ -226,54 +266,62 @@ Analytics.getVisitorIdentifier(object: AdobeCallbackWithError<String> {
...
-
+##### Example
 ```Java
 Analytics.sendQueuedHits();
 ```
 
 #### Kotlin
 
+##### Syntax
+```Kotlin
```
Comment (addb) [inline Documentation/api-reference.md:288]: small k
` ```kotlin`

File: Documentation/api-reference.md [hunk 13]
```diff
@@ -226,54 +266,62 @@ Analytics.getVisitorIdentifier(object: AdobeCallbackWithError<String> {
...
 ```Kotlin
 Analytics.sendQueuedHits()
 ```
 
 ---
 
-## setVisitorIdentifier
+### setVisitorIdentifier
 
 > Before use this API, see [Identify unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html).
```
Comment (addb) [inline Documentation/api-reference.md:301]: Add note line as mentioned above

File: Documentation/api-reference.md [hunk 14]
```diff
@@ -226,54 +266,62 @@ Analytics.getVisitorIdentifier(object: AdobeCallbackWithError<String> {
...
 
 Sets a custom Analytics visitor identifier. For more information, see [Custom Visitor ID](https://experienceleague.adobe.com/docs/analytics/implementation/vars/config-vars/visitorid.html).
 
-### Syntax
+#### Java
 
+##### Syntax
 ```Java
```
Comment (addb) [inline Documentation/api-reference.md:308]: small j

` ```java ` for all java code block formatting

File: Documentation/api-reference.md [hunk 15]
```diff
@@ -212,6 +247,11 @@ Analytics.getVisitorIdentifier(new AdobeCallbackWithError<String>() {
...
 #### Kotlin
 
+##### Syntax
+```Kotlin
+ fun getVisitorIdentifier(callback: AdobeCallback<String?>) 
+```
+##### Example
 ```Kotlin
 Analytics.getVisitorIdentifier(object: AdobeCallbackWithError<String> {
     override fun call(id: String?) {
```
Comment (addb) [inline Documentation/api-reference.md:257]: `override fun call(vid: String?)`

File: Documentation/api-reference.md [hunk 16]
```diff
@@ -180,22 +217,20 @@ Analytics.getTrackingIdentifier(object: AdobeCallbackWithError<String> {
...
 ```
 
-### Example
-
-#### Java
-
+##### Example
 ```Java
 Analytics.getVisitorIdentifier(new AdobeCallbackWithError<String>() {
     @Override
```
Comment (addb) [inline Documentation/api-reference.md:236]: `public void call(String vid) {`

File: README.md [hunk 17]
```diff
@@ -1,4 +1,4 @@
-# Adobe Analytics Android Extension
+# Adobe Experience Platform - Analytics extension for Android
```
Comment (addb) [inline README.md:1]: we can keep`Extension` instead of `extension` to match all our headers in other files

File: Documentation/README.md [hunk 18]
```diff
@@ -1,4 +1,4 @@
-# Adobe Analytics Extension Documentation
+# Analytics extension documentatioin
```
Comment (addb) [inline Documentation/README.md:1]: Adobe Experience Platform - Analytics Extension for Android Documentation

File: Documentation/README.md (hunk 2 above)
Comment (cacheung) [reply Documentation/README.md]: I followed Media to use "Analytics extension documentation".

File: Documentation/api-reference.md [hunk 19]
```diff
@@ -1,79 +1,100 @@
 # Analytics API Usage
```
Comment (addb) [inline Documentation/api-reference.md:1]: API Reference

File: Documentation/api-reference.md [hunk 20]
```diff
@@ -1,79 +1,100 @@
...
+- [extensionVersion](#extensionVersion)
+- [EXTENSION](#EXTENSION)
+- [getQueueSize](#getQueueSize)
+- [getTrackingIdentifier](#getTrackingIdentifier)
+- [getVisitorIdentifier](#getVisitorIdentifier)
+- [sendQueuedHits](#sendQueuedHits)
+- [setVisitorIdentifier](#setVisitorIdentifier)
+------
+
+
```
Comment (addb) [inline Documentation/api-reference.md:19]: move L20 `This document details all the APIs provided by Analytics extension, along with sample code snippets on how to properly use the APIs.` to before the API list i.e on L9

File: Documentation/api-reference.md [hunk 21]
```diff
@@ -180,41 +218,45 @@ Analytics.getTrackingIdentifier(object: AdobeCallbackWithError<String> {
 
 ---
 
-## getVisitorIdentifier
+### getVisitorIdentifier
 
-> Before use this API, see [Identify unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html).
+> **Note**
+Before use this API, see [Identify unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html).
```
Comment (emdobrin) [inline Documentation/api-reference.md:224]: Before using this API

File: Documentation/api-reference.md [hunk 22]
```diff
@@ -82,6 +103,13 @@ MobileCore.registerExtensions(Arrays.asList(Analytics.EXTENSION, ...), new Adobe
 
 #### Kotlin
 
+##### Syntax
+```kotlin
+val EXTENSION: Class<out Extension?> = AnalyticsExtension::class.java
+}
```
Comment (addb) [inline Documentation/api-reference.md:109]: remove }

File: Documentation/api-reference.md (hunk 21 above)
Comment (addb) [inline Documentation/api-reference.md:224]: Before `using` this ...

File: Documentation/api-reference.md [hunk 23]
```diff
@@ -226,54 +268,63 @@ Analytics.getVisitorIdentifier(object: AdobeCallbackWithError<String> {
...
 ```
 
 ---
 
-## setVisitorIdentifier
+### setVisitorIdentifier
 
-> Before use this API, see [Identify unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html).
+> **Note** 
+Before use this API, see [Identify unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html).
```
Comment (addb) [inline Documentation/api-reference.md:304]: Before using this ...

**Review by emdobrin — APPROVED** [review]

**Review by addb — APPROVED** [review]

**Pull request merged**